arcgis-ios-sdk-samples/Scenes/Add a point scene layer/AddPointSceneLayer.storyboard

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14460.31" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="QqC-8h-4uR">
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14460.20"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Add Point Scene Layer View Controller-->
+        <scene sceneID="i0f-CX-ZXN">
+            <objects>
+                <viewController id="QqC-8h-4uR" customClass="AddPointSceneLayerViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="Rl5-tq-Yyk">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="O4F-p8-iOE" customClass="AGSSceneView">
+                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
+                                <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                            </view>
+                        </subviews>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstItem="O4F-p8-iOE" firstAttribute="bottom" secondItem="sNx-8h-g47" secondAttribute="bottom" id="bVL-4n-p33"/>
```

Comment:
Please update the bottom, leading, and trailing constraints to the superview rather than the safe area.

![Screen Shot 2019-03-19 at 9 12 34 AM](https://user-images.githubusercontent.com/2257493/54622641-2a937e80-4a27-11e9-8eb6-1246a4ef1965.png)

Reply:
You don't need to recreate the constraints. You can simply update the appropriate item.

![Screen Shot 2019-03-19 at 9 22 21 AM](https://user-images.githubusercontent.com/2257493/54623355-97f3df00-4a28-11e9-9695-23d3e27c0b33.png)

Reply:
Sorry missed this form factor and orientation. Updated the constraints